Tribler/Test/test_as_server.py

```diff
@@ -301,6 +302,7 @@ class TestGuiAsServer(TestAsServer):
 
     def setUp(self):
         AbstractServer.setUp(self, annotate=False)
+        self.setUpPreSession()
```

Comment:
@NielsZeilemaker This is only for `test_remote_search` because it needs to create a SessionConfig, set some options and pass it to `tribler_main`. The line doesn't have any effect on other tests because the other GUI tests are simply running full Tribler. In `test_remote_search`, the config is the same as full Tribler except `torrent_search` and `channel_search`, so we are still testing full Tribler, but for `torrent_seach` only the `SearchCommunity` is enabled and for `channel_seach` the `AllChannelCommunity`.